.gitignore

```diff
@@ -135,4 +135,7 @@ dmypy.json
 temp/
 
 # MAC
-.DS_Store
+.DS_Stor
```

Comment:
Weird:  Mac creates files called ".DS_Store" but I've never seen ".DS_Stor".

Reply:
i am inflicted with this problem https://buildthis.com/ds_store-files-and-why-you-should-know-about-them/

Reply:
Yes, the article you linked talks about ".DS_Store" files, which I'm familiar with.  But this PR removes ".DS_Store" from .gitignore (and adds ".DS_Stor").  Shouldn't we include .DS_Store" in .gitignore?

Reply:
ahh i see.  i think that was accidental.  i have restored it. thanks.